<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QWidget" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Principal</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seleccione la opcion que desea analizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IrSucursal">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KacstLetter</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sucursales</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IrCliente">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KacstLetter</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IrVenta">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KacstLetter</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ventas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Salir">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KacstLetter</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KacstLetter</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
